_analyzers/token-filters/asciifolding.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: ASCIIFolding
+parent: Token filters
+nav_order: 20
+---
+
+# ASCIIFolding token filter
+
+`asciifolding` is a token filter that converts non-ASCII characters into their closest ASCII equivalents. For example *é* becomes *e*, *ü* becomes *u* and *ñ* becomes *n*. This process is also known as *transliteration*.
```

Comment:
Normally, we want to avoid starting sentences with words in code font. Here's an alternative:
```suggestion
The `asciifolding` oken filter converts non-ASCII characters into their closest ASCII equivalents. For example *é* becomes *e*, *ü* becomes *u* and *ñ* becomes *n*. This process is known as *transliteration*.
```